core/catppuccin-jsonresume-theme/template/cv.css

```
@@ -0,0 +1,584 @@
+		color: #111 !important;
+	}
+	.no-print {
+		display: none !important;
+	}
+	.cv-container {
+		padding: 0.5in 0.75in;
+		max-width: none;
+	}
+	.cv-section {
+		margin-top: 2rem;
+		page-break-inside: avoid;
```

Stylelint (17.12.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)